src/main/scala/algolia/AlgoliaUtils.scala

```diff
@@ -25,12 +25,16 @@
 
 package algolia
 
-import scala.util.{Random => ScalaRandom}
+import java.util.Date
 
-private[algolia] class AlgoliaRandom {
+import scala.util.Random
 
-  def shuffle(seq: Seq[String]) = ScalaRandom.shuffle(seq)
+private[algolia] class AlgoliaUtils {
+
+  def shuffle(seq: Seq[String]): Seq[String] = Random.shuffle(seq)
```

Comment:
Can't find the information online so I'll ask: don't you need to set a seed to initialize the PRNG behind random or is it already done for you?